Wenab/Wenab.Api/Program.cs

```diff
@@ -1,25 +1,29 @@
-var builder = WebApplication.CreateBuilder(args);
+using LittleByte.Common;
+using LittleByte.Common.AspNet.Middleware;
+using LittleByte.Common.AspNet.Configuration;
+using LittleByte.Common.Configuration;
+using Wenab.Ynab;
 
-// Add services to the container.
+var builder = WebApplication.CreateBuilder(args);
 
 builder.Services.AddControllers();
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
-builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
-
-var app = builder.Build();
-
-// Configure the HTTP request pipeline.
-if (app.Environment.IsDevelopment())
-{
-    app.UseSwagger();
-    app.UseSwaggerUI();
-}
+builder.Services
+    .AddOpenApi("Wenab")
+    .AddSingleton<IDateService, DateService>()
+    .AddEndpointsApiExplorer()
+    .AddSwaggerGen();
 
-app.UseHttpsRedirection();
+builder.Services.BindOptions<YnabOptions>(builder.Configuration);
 
-app.UseAuthorization();
+var app = builder.Build();
 
-app.MapControllers();
+app
+    .UseHttpsRedirection()
+    .UseHsts()
```

Comment:
`UseHsts`?

Reply:
HSTS essentially forces users to interact with the service with HTTPS. It's a security feature that we can just enable and not really think about.